main/src/examples/ABMAugmentionExample/ABMAugmentionExample.cpp

```diff
@@ -187,22 +191,16 @@ void ABMAugmentionExample::augmentImages() {
         //imshow("Canny", augmented);
         //waitKey(50);
 
-        // convert back to IplImage
-        IplImage augmentedImageIpl = augmented;
-        // end your stuff
-        // from IplImage to yarp image
-        cvCvtColor(&augmentedImageIpl, &augmentedImageIpl, CV_BGR2RGB);
         ImageOf<PixelRgb> augmentedImageYarp;
-        augmentedImageYarp.resize(augmentedImageIpl.width, augmentedImageIpl.height);
-        cvCopyImage(&augmentedImageIpl, (IplImage *)augmentedImageYarp.getIplImage());
+        augmentedImageYarp.resize(vRawImages[i]);
+        Mat img3_mat=cvarrToMat((IplImage*)augmentedImageYarp.getIplImage());
+        cvtColor( augmented, img3_mat, CV_BGR2RGB );
```

Comment:
I think here you change the logic, do you? Previously, the color conversion happened first, and then the `IplImage` was converted to the yarp image format. Now, you convert `augmentedImageYarp` to a `cv::Mat`, but `augmentedImageYarp` is empty. Then, you convert the color space of the `cv::Mat` object, and then just disregard it. Or am I wrong?